examples/simulation_pytorch/dataset.py

```diff
@@ -0,0 +1,78 @@
+"""Partitioned version of CIFAR-10 dataset."""
+
+from typing import List, Tuple, cast
+
+import numpy as np
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader, TensorDataset
+from numpy import ndarray
+
+XY = Tuple[np.ndarray, np.ndarray]
+XYList = List[XY]
+PartitionedDataset = List[Tuple[XY, XY]]
+
+
+def cifar_to_numpy() -> Tuple[Tuple[ndarray, ndarray], Tuple[ndarray, ndarray]]:
```

Comment:
```suggestion
def cifar_to_numpy() -> Tuple[XY, XY]:
```

Reply:
A side note do you generate type hints manually or use some automated tool. Thanks. @tanertopal